### PR TITLE
Example of using ethers for making transactions with web3-onboard

### PR DESCRIPTION
**Pull request by meshin-dev**

### CODE REVIEW

File: src/lib/utils/onboardHelper.ts [hunk 1]
```diff
@@ -13,11 +14,16 @@ export const initOnboard = () => {
     optionalChains: [NETWORKS.MAINNET, NETWORKS.GOERLI, NETWORKS.HOLESKY],
   });
   const theme = window.localStorage.getItem('isDarkMode') === '1' ? 'dark' : 'light';
+  const safeWalletInstance = safeWallet();
 
   return Onboard({
     theme: theme,
     apiKey: config.ONBOARD.API_KEY,
-    wallets: [injected, walletConnect],
+    wallets: [
```
Comment (IlyaVi) [inline src/lib/utils/onboardHelper.ts:22]: add here the code I've sent you

File: src/app/common/stores/applications/SsvWeb/Account.store.ts [hunk 2]
```diff
@@ -1,71 +1,68 @@
...
+  async setFeeRecipient(feeRecipient: string): Promise<boolean> {
+    return new Promise<boolean>(async (resolve) => {
+      const applicationStore: ApplicationStore = this.getStore('Application');
+      const walletStore: WalletStore = this.getStore('Wallet');
+      const contract: Contract = walletStore.setterContract;
+      try {
+        const tx = await contract.setFeeRecipientAddress(feeRecipient);
+        if (tx.hash) {
+          applicationStore.txHash = tx.hash;
+          applicationStore.showTransactionPendingPopUp(true);
```
[the remark this answers is not in the document]
Comment (meshin-dev) [reply src/app/common/stores/applications/SsvWeb/Account.store.ts:29]: In all the other places where you read tx.hash do the saem pls

**Review by IlyaVi — APPROVED** [review]

File: src/app/common/stores/applications/SsvWeb/Account.store.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (axelrod-blox) [reply src/app/common/stores/applications/SsvWeb/Account.store.ts:29]: done

[the remark this answers is not in the document]
Comment (axelrod-blox) [reply src/app/common/stores/applications/SsvWeb/Account.store.ts:31]: if we have any error here, reject or contract return error we came to catch block

**Pull request merged**